DevTools/ClientHints/explainer.md

```diff
@@ -0,0 +1,53 @@
+# Edit Client Hint headers 
+
+Authors: [Erica Draud](https://github.com/erdraud)
+
+## Status of this Document
+* This document status: **Active**
+* **Current version: this document**
+    
+## Introduction
```

Comment:
I personally didn't know about Client Hints (let alone UA Client Hints). I'm guessing it isn't a very widely known API yet. It might be good to add a short paragraph explaining what Client Hints are in general first, before going into the specifics of UA Client Hints.

Reply:
Thanks, just added some links and context for this.